apps/crn-server/src/utils/odata.ts

```diff
@@ -0,0 +1,48 @@
+import buildQuery, { Filter } from 'odata-query';
+import { ResearchOutputFilter } from '../types';
+import { getFirstOrAll } from './arrays';
+
+const FILTER_URL_QUERY_PARAM = '?$filter=';
+
+export const makeODataFilter = (
+  filter?: ResearchOutputFilter,
```

Comment:
this clearly isn't a util if it's taking a very specific type as input. You may want to move this next to the data provider